config/yurtadm/cloudimage/openyurt-latest/yamls/yurt-controller-manager.yaml

```diff
@@ -0,0 +1,148 @@
+apiVersion: v1
```

Comment:
how about use yaml file in `openyurtio/openyurt/config/setup` directory? so we don't need to manage yaml file of openyurt here.

Reply:
@rambohe-ch 
At present, some parameters in yaml are not fixed, but are rendered by env in clusterfile, that is, the suffix `.yaml.tmpl` file. Later, if you need to add user-defined parameters in env, `.yaml.tmpl` files also need to be changed, so the file here (`.yaml.tmpl` ) may be somewhat different from that in the `openyurtio/openyurt/config/setup` directory.


see: [Using Clusterfile to init a cluster - Env render support | sealer](http://sealer.cool/docs/getting-started/using-clusterfile.html#env-render-support)